Dockerfile.production

```diff
@@ -6,6 +6,8 @@ COPY package.json package-lock.json /usr/src/app/
 
 RUN npm ci
 
+ARG SENTRY_DSN
```

Comment:
It seems the `SENTRY_DSN` is needed when building webpack dist, not sure if putting it here is the best way @inDream @rickmak

Reply:
Yes, ant put the value into docker-compose.yml

```
build:
  context: .
  args:
    SENTRY_DSN: XXX
```

Reply:
updated and also set the DSN as ENV in Dockerfile, since default server DSN aligns with client build seems more sensible